src/Compilers/CSharp/Test/Emit3/RefReadonlyParameterTests.cs

```diff
@@ -2146,15 +2146,12 @@ static class C
             """;
         var expectedDiagnostics = new[]
         {
-            // (3,30): error CS0246: The type or namespace name 'scoped' could not be found (are you missing a using directive or an assembly reference?)
+            // (3,30): error CS9347: The 'scoped' modifier cannot come after an 'in', 'out', 'ref' or 'readonly' modifier.
             //     public static void M(ref scoped readonly int p) => throw null;
-            Diagnostic(ErrorCode.ERR_SingleTypeNameNotFound, "scoped").WithArguments("scoped").WithLocation(3, 30),
-            // (3,37): error CS1001: Identifier expected
+            Diagnostic(ErrorCode.ERR_ScopedAfterInOutRefReadonly, "scoped").WithLocation(3, 30),
+            // (3,37): error CS9348: The 'readonly' modifier cannot immediately follow the 'scoped' modifier.
             //     public static void M(ref scoped readonly int p) => throw null;
-            Diagnostic(ErrorCode.ERR_IdentifierExpected, "readonly").WithLocation(3, 37),
-            // (3,37): error CS1003: Syntax error, ',' expected
-            //     public static void M(ref scoped readonly int p) => throw null;
-            Diagnostic(ErrorCode.ERR_SyntaxError, "readonly").WithArguments(",").WithLocation(3, 37),
```

Comment:
lots of the PR will just be syntax errors going away, and better errors about misplaced 'scopes' modifiers.